### PR TITLE
[lcc_members_portal] Account email change  also modify user's login

### LINKED ISSUE #151: [lcc members portal] fix bug of user mail not change from portal
Lorsqu'on change son adresse email depuis les préférences utilisateur portal ça change uniquement l'adresse email de la fiche contact et non du compte utilisateur. Je ne comprends pas qu'on ne puisse pas changer son adresse email et son mdp quand on est un utilisateur portail.

pour 10€ on a ce module qui est en licence LGPL-3 : https://apps.odoo.com/apps/modules/12.0/se_website_change_password/

Point de vigilance: il faut qu'on voit si le changement de l'email de Odoo ne pose pas de pb par rapport au SSO : le changement de mail dans Odoo ne génère pas de changements dans Rocketchat ou Nextcloud, du coup est-ce que le matching des comptes fonctionne toujours ?

### PR DESCRIPTION
**Pull request by stephansainleger**

### CODE REVIEW

File: lcc_members_portal/controllers/custom_portal.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Lokavaluto ()
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.http import request, route
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+
+class CustomCustomerPortal(CustomerPortal):
+    @route(["/my/account"], type="http", auth="user", website=True)
+    def account(self, redirect=None, **post):
+        response = super(CustomCustomerPortal, self).account(redirect, **post)
```
Comment (njeudy) [inline lcc_members_portal/controllers/custom_portal.py:10]: We need to test the unicity and if user.login can be different from email (ex admin account) .. think email should not be considered as only way to login .

**Comment by njeudy** [comment]
I think we need to send en new reset password to validate the email in this case ?

**Review by njeudy — CHANGES_REQUESTED** [review]
We need to check new email is not already used.

**Comment by stephansainleger** [comment]
> 

check of email uniqueness already coded and delivered in PR https://github.com/Lokavaluto/lokavaluto-addons/pull/167

File: lcc_members_portal/controllers/custom_portal.py (hunk 1 above)
Comment (stephansainleger) [reply lcc_members_portal/controllers/custom_portal.py:10]: Check of email uniqueness already coded and delivered in PR https://github.com/Lokavaluto/lokavaluto-addons/pull/167

For the user.login, technically we still can change the login from Odoo Backend user form view (and put an non-email login), but I would prefer to not separate login and email for "standard" user, to simplify the processes and forms.

@MatSarr : do you have an opinion on that point?

Comment (MatSarr) [reply lcc_members_portal/controllers/custom_portal.py:10]: I agree with Stéph, the UX is already quite intricate, I would like it to be as smooth as possible. So I reckon the login should be the email for the user.

**Review by njeudy — APPROVED** [review]

**Pull request merged**